vars/main.yml

```diff
@@ -8,3 +8,4 @@ java_release: "212"
 java_name: "jdk-8u212-macosx-x64"
 java_url: "https://github.com/frekele/oracle-java/releases/download/{{ java_version }}u{{ java_release }}-{{ java_build_version }}/{{ java_name }}.dmg"
 sdk_buildtools_version: "27.0.3"
+nvm_node_version: "v10.19.0"
```

Comment:
nvm or node?
Should be named as `node_version` or `nodejs_version`
But it's not necessary.